packages/TorneloScoresheet/src/components/Toolbar/Toolbar.tsx

```diff
@@ -142,7 +142,7 @@ const Toolbar: React.FC = () => {
         )}
         {showPlayerModeButton && (
           <IconButton
-            icon="lock"
+            icon="lock-open"
```

Comment:
I'm thinking it might make more sense to invert these actually,
When you are in player mode, you want to 'unlock' the app into Arbiter mode.
When you're in Arbiter mode, you want to 'lock' the app back into player mode?
What do you think?